types/materialize-css/datepicker.d.ts

```diff
@@ -88,6 +88,7 @@ declare namespace M {
 
         /**
          * The initial date to view when first opened.
+         * @default null
          */
         defaultDate: Date;
```

Comment:
How about just declare it as nullable here?

```suggestion
        defaultDate: Date | null;
```

Reply:
Added to the PR when switching to explicit `| null` annotations - no longer applicable